src/features/Settings/Settings.tsx

```diff
@@ -49,6 +55,65 @@ const StyledLayout = styled.div`
   }
 `;
 
+function rendedExcludedAssetsPanel() {
+  const { accounts, totals, currentAccounts } = useContext(StoreContext);
+  const { settings, removeAssetfromExclusionList } = useContext(SettingsContext);
+  const { getAssetRate } = useContext(RatesContext);
+
+  const balances: Balance[] = totals(currentAccounts)
+    .filter((asset: StoreAsset) => {
+      return !isExcludedAsset(settings.excludedAssets)(asset);
+    })
+    .map((asset: StoreAsset) => {
+      const exchangeRate = getAssetRate(asset);
+      return {
```

Comment:
Use existing utility methods for all of this if possible or abstract them out from `StoreProvider`

Reply:
I'm fairly sure we have existing logic for this.

Reply:
I extracted it out so its not duplicated anymore c: